packages/app/obojobo-document-engine/src/scripts/oboeditor/components/toolbars/list-dropper.js

```diff
@@ -93,56 +93,80 @@ class ListDropper extends React.Component {
 	}
 
 	toggleBullet() {
-		const nodes = Array.from(Editor.nodes(this.props.editor, {
-			mode: 'lowest',
-			match: node => Element.isElement(node) && !this.props.editor.isInline(node) && !node.subtype
-		}))
+		const nodes = Array.from(
+			Editor.nodes(this.props.editor, {
+				mode: 'lowest',
+				match: node => Element.isElement(node) && !this.props.editor.isInline(node) && !node.subtype
+			})
+		)
 		const isList = nodes.every(([block]) => block.type === LIST_NODE)
 
-		if(!isList) {
-			return this.props.editor.changeToType(LIST_NODE, { type: this.props.type, bulletStyle: this.props.defaultStyle })
+		if (!isList) {
+			return this.props.editor.changeToType(LIST_NODE, {
+				type: this.props.type,
+				bulletStyle: this.props.defaultStyle
+			})
 		}
 
 		// Once we know they are all lists, we can check if the lists are the same type
 		// as the list we are changing to
 		const isSameType = nodes.every(([block]) => block.content.listStyles.type === this.props.type)
-		if(!isSameType) {
-			return this.props.editor.changeToType(LIST_NODE, { type: this.props.type, bulletStyle: this.props.defaultStyle })
+		if (!isSameType) {
+			return this.props.editor.changeToType(LIST_NODE, {
+				type: this.props.type,
+				bulletStyle: this.props.defaultStyle
+			})
 		}
 
 		this.props.editor.changeToType(TEXT_NODE)
 	}
 
 	render() {
+		const isMac = navigator.appVersion.indexOf('Mac') !== -1
+		// Decide whether or not to use the mac shortcut
+		// Note - users can spoof their appVersion, but anyone who is tech-savvy enough
+		// to do that is probably tech-savvy enough to know whether they use CTRL or ⌘
+		// for keyboard shortcuts
+		const shortcutMac = isMac && this.props.shortcutMac ? '\n' + this.props.shortcutMac : ''
+		// If the Mac shortcut exists, use it
+		// If there is no Mac shortcut and no mark.shortcut, the mac shortcut will be
+		// the blank string, so just use it
+		const shortcut = shortcutMac || !this.props.shortcut ? shortcutMac : '\n' + this.props.shortcut
```

Comment:
The only place where we'd have a problem with this is in Format > Paragraph styles; I've set the Mac shortcuts to CTRL+SHIFT+1...6, because Cmd+Shift+3..5 are screenshot shortcuts.  But everything else has been changed to modifier + keys